docs/false_positives.md

```diff
@@ -130,6 +131,20 @@ Other macros or builtins expressing unreachable code may be used.
 Note that the rewritten code is also safer, since debug builds now check for
 more precondition violations. 
 
+In case of C++11 or later, another option is to use immediately evaluated
+function expression (IEFE) to avoid assigning a meaningless value.
```

Comment:
Googling this expression does not seem to throw up anything. Is it possible that you meant [Immeadiately-Invoked Function Expression (IIFE)](https://cppnow2015.sched.com/event/2np7/iife-in-c-for-performance-and-safety), which seems to be a widely used term?